javalib/src/main/scala/java/util/concurrent/LinkedTransferQueue.scala

```diff
@@ -0,0 +1,1804 @@
+/*
+ * Written by Doug Lea with assistance from members of JCP JSR-166
+ * Expert Group and released to the public domain, as explained at
+ * http://creativecommons.org/publicdomain/zero/1.0/
+ */
+
+package java.util.concurrent
+
+import java.util.{AbstractQueue, Collection, Iterator, Spliterator, function}
+import java.util.{Arrays, Objects, Spliterators}
+import java.util.{NoSuchElementException}
+import java.util.concurrent.atomic.AtomicReference
+import java.util.concurrent.locks.LockSupport
+import scala.scalanative.libc.stdatomic.AtomicRef
+import scala.scalanative.libc.stdatomic.memory_order.{
+  memory_order_relaxed,
+  memory_order_release
+}
+import scala.scalanative.runtime.{fromRawPtr, Intrinsics}
+
+/** An unbounded {@link TransferQueue} based on linked nodes. This queue orders
```

Comment:
You can drop the scaladoc comments, these are problematic when using `{ x }` javadoc syntax which might be splitted by scalafmt and moved partially to next line making them invalid. Because of that Scala 2 scaladoc fails the compilation. In case of javalib we use scaladoc only to have a database of implemented methods/classes. The user shall refer to JDK scaladoc instead for details.